src/Graphics/Effect/EffectAnnotation.cs

```diff
@@ -141,11 +142,7 @@ public float GetValueSingle()
 
 		public string GetValueString()
 		{
-			/* FIXME: This requires digging into the effect->objects list.
-			 * We've got the data, we just need to hook it up to FNA.
-			 * -flibit
-			 */
-			throw new NotImplementedException("effect->objects[?]");
+			return cachedString ?? String.Empty;
```

Comment:
We _may_ need to double check the EffectParameter Type here, otherwise it can just be
```suggestion
			return cachedString;
```